src/application.rs

```diff
@@ -113,6 +118,12 @@ impl Application for RusticApp {
 impl RusticApp {
     /// Shut down this application gracefully, exiting with given exit code.
     fn shutdown_with_exitcode(&self, shutdown: Shutdown, exit_code: i32) -> ! {
```

Comment:
We're not using that throughout the commands, right? So these hooks will not be run, I assume?

Reply:
Ah no, we always call it, didn't see the changes in `Application::shutdown`